src/app/datagrid/selection/selection.html

```diff
@@ -46,7 +46,11 @@ <h3>Client side, no trackBy</h3>
     <ng-container *clrDgHideableColumn> Favorite color </ng-container>
   </clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let user of clientNoTrackByUsers" [clrDgItem]="user">
+  <clr-dg-row
+    *clrDgItems="let user of clientNoTrackByUsers"
+    [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.